samples/Azure.ResourceManager.Sample/Generated/AvailabilitySet.cs

```diff
@@ -152,7 +152,7 @@ public virtual IEnumerable<Location> GetAvailableLocations(CancellationToken can
         /// <summary> Delete an availability set. </summary>
         /// <param name="waitForCompletion"> Waits for the completion of the long running operations. </param>
         /// <param name="cancellationToken"> The cancellation token to use. </param>
-        public async virtual Task<AvailabilitySetDeleteOperation> DeleteAsync(bool waitForCompletion = true, CancellationToken cancellationToken = default)
```

Comment:
not final on this as the approach lets hold off on merging until we get confirmation